docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+---
+title: "Path algorithms"
+linkTitle: "Path algorithms"
+description: "Learn how to use algo.SPpaths and algo.SSpaths to solve queries"
```

Comment:
`"to solve queries"`
This description is a bit confusing, `queries` so much too broader